code/modules/mining/abandoned_crates.dm

```diff
@@ -55,8 +59,9 @@
 				attempts--
 				if(attempts == 0)
 					boom(user)
-	else
-		return ..()
+		return
+
+	. = ..()
```

Comment:
why?

Reply:
return ..() is easier to read, it makes more sense and theres no reason to switch it to . = ..(), why did you change it and not give it any explanation

Reply:
if this is justified, change this to `return ..()` but keep the else

Reply:
I was told by Gooncoders that it looks better smh. Will change it.

Reply:
I love Goon but style guidelines are not something I'd take from them :fire: